File: ext/RmlUI.Net/StyleSheetSpecification.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RmlUiNet
{
    public static class StyleSheetSpecification
    {
        public static PropertyDefinition RegisterProperty(string name, string defaultValue, bool inherited, bool forcesLayout = false) {
            var intPtr = RmlUiNet.Native.StyleSheetSpecification.RegisterProperty(name, defaultValue, inherited, forcesLayout);
            return new PropertyDefinition(intPtr);
        }
    }
}

File: ext/RmlUi.Native/src/StyleSheetSpecification.cpp
#include "RmlNative.h"
#include "RmlUi/Core.h"
#include "RmlUi/Core/StyleSheetSpecification.h"
#include "Pool.h"

Rml::PropertyDefinition* def;

RMLUI_CAPI Rml::PropertyDefinition* rml_StyleSheetSpecification_RegisterProperty(const char* property_name, const char* default_value, bool inherited, bool forces_layout) {
  def = &Rml::StyleSheetSpecification::RegisterProperty(property_name, default_value, inherited, forces_layout);
  return def;
}